core/domain/user_domain.py

```diff
@@ -316,13 +316,15 @@ class UserContributionRights(python_utils.OBJECT):
 
     def __init__(
             self, user_id, can_review_translation_for_language_codes,
-            can_review_voiceover_for_language_codes, can_review_questions):
+            can_review_voiceover_for_language_codes, can_review_questions,
+            can_submit_questions=False):
```

Comment:
Why is this False by default?

Reply:
Agreed--this feels a bit unexpected since ``can_review_questions`` & other fields aren't defaulted.